dragonchain/broadcast_processor/broadcast_processor.py

```diff
@@ -110,6 +120,81 @@ def make_broadcast_futures(session: aiohttp.ClientSession, block_id: str, level:
     return broadcasts
 
 
+def get_level_from_storage_location(storage_location: str) -> Optional[str]:
+    result = re.search("BLOCK/.*?-l([2-5])-", storage_location)
+    if result is None:
+        return None
+    return result.group(1)
+
+
+def get_notification_urls(key: str) -> set:
+    try:
+        urls = set(VERIFICATION_NOTIFICATION[key])
+    except KeyError:
+        urls = set()
+    return urls
+
+
+def sign(message: bytes) -> str:
+    return keys.get_my_keys().make_signature(message, crypto.SupportedHashes.sha256)
+
+
+def get_all_notification_endpoints(level: str) -> set:
+    return get_notification_urls("all").union(get_notification_urls(f"l{level}"))
+
+
+async def process_verification_notifications(session: aiohttp.ClientSession) -> None:
+    """Main function for the verification notification broadcast system
+
+    Retrieves verifications that need to be processed and sends http requests
+
+    Args:
+        session: aiohttp session for http requests
+    """
+    if VERIFICATION_NOTIFICATION:
+        futures = set()
+        for storage_location in await broadcast_functions.get_notification_verifications_for_broadcast_async():
+            verification_bytes = storage.get(storage_location)
+            level = get_level_from_storage_location(storage_location)
+            if level is None:
+                _log.error(f"Unable to parse level value from string {storage_location}. Removing verification notification from set.")
+                broadcast_functions.remove_notification_verification_for_broadcast_async(storage_location)
+                continue
+            signature = sign(verification_bytes)
+            for url in get_all_notification_endpoints(level):
+                future = send_notification_verification(session, url, verification_bytes, signature, storage_location)
+                futures.add(asyncio.ensure_future(future))
+        try:
+            await asyncio.gather(*futures, return_exceptions=True)  # "Fire & forget"
+        except Exception:
```

Comment:
you don't need this try/except if you have return_exceptions=True

asyncio.gather should never throw an exception in this configuration unless something is seriously wrong